custom/enikshay/ucr/data_sources/test_2b_v2.json

```diff
@@ -0,0 +1,3377 @@
+{
+    "domains": [
+        "enikshay",
```

Comment:
this is called "test_2b_v2" If it's in testing does it need to be in the enikshay domain, or can it be kept in the QA domain for now and added to enikshay later?

Reply:
its needs to run on the real data. so should be under enikshay?

Reply:
it's the `test` case type :smile:

Reply:
oh ya.. test is the case type :)